src/modules/auth/auth.service.ts

```
@@ -423,4 +460,37 @@ export class AuthService {
       });
     }
   }
+
+  public async updatePassword(body: UpdatePasswordDTO) {
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)